src/world.cpp

```diff
@@ -331,7 +327,7 @@ void World::reset_game() {
 
 	m_lane_manager.reset();
 	m_game_timer.reset();
-	m_level_manager.init(); // only sets primitives, no memory leak
+	m_level_manager.init(m_high_scores); // only sets primitives, no memory leak

```

Comment:
I think this is okay because it doesn't actually initialize anything in the init function